mne/preprocessing/nirs/_beer_lambert_law.py

```diff
@@ -38,10 +38,13 @@ def beer_lambert_law(raw, ppf=0.1):
     picks = _check_channels_ordered(raw.info, freqs)
     abs_coef = _load_absorption(freqs)
     distances = source_detector_distances(raw.info)
+    if (distances == 0).any():
+        warn('Source-detector distances are zero, some resulting '
+             'concentrations will be zero. Consider setting a montage '
+             'with raw.set_montage.')
 
```

Comment:
I think we should add a check for distances which are too large too. fNIRS channels should be 3ish cm distance so that enough light is received by the detector. Could we throw a warning if the distance is greater than an average head? So something like 8-10cm? This would catch the situations where people are storing the distances as cm rather than m. Just a thought

```suggestion
if (distances > 0.1).any():
    warn('Source-detector distances are greater than 10 cm. '
              'Large distances will result in invalid data, and are '
              'likely due to optode locations being stored in a '
              ' unit other than meters.')
```